### PR TITLE
Bugfix/missing seo titles

### PR DESCRIPTION
**Pull request by suejinkim20**
This PR addresses issue #66

### CODE REVIEW

File: src/templates/events/past-events-template.jsx [hunk 1]
```diff
@@ -15,9 +15,7 @@ const PastEventsTemplate = ({data}) => {
   })
 
   return (
-    <PageContent width="95%" maxWidth="800px">
-
-      <Title>Event Archive</Title>
+    <PageContent width="95%" maxWidth="800px" title="Event Archive">
```
Comment (mbwatson) [inline src/templates/events/past-events-template.jsx:18]: `width="95"` is the default. let's remove unnecessary props like this.

File: src/templates/events/upcoming-events-template.jsx [hunk 2]
```diff
@@ -19,9 +16,7 @@ const UpcomingEventsList = ({ data }) => {
   })
 
   return (
-    <PageContent width="95%" maxWidth="800px">
-
-      <Title>Upcoming Events</Title>
+    <PageContent width="95%" maxWidth="800px" title="Upcoming Events">
```
Comment (mbwatson) [inline src/templates/events/upcoming-events-template.jsx:19]: same as above

File: src/pages/use-bdc/explore-data/dug/index.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const DugSearchPage = () => {
   }, [sidebarVisibility])
 
   return (
-    <PageContent>
+    <PageContent hideTitle title="Disease, Phenotype, Biological Process, or Anatomical Entity Search">
```
Comment (mbwatson) [inline src/pages/use-bdc/explore-data/dug/index.js:28]: :+1:

File: src/templates/tag-template.jsx [hunk 4]
```diff
@@ -12,7 +12,7 @@ const TagTemplate = ({ data, pageContext }) => {
   const events = data.events.nodes;
 
   return (
-    <PageContent maxWidth="1000px">
+    <PageContent maxWidth="1000px" title={`${tag} - Tagged`} hideTitle>
```
Comment (mbwatson) [inline src/templates/tag-template.jsx:15]: :+1:

File: src/templates/latest-updates/index.jsx [hunk 5]
```diff
@@ -18,8 +18,7 @@ const LatestUpdatesPost = ({ data: { mdx }, pageContext, children }) => {
   const { prev, next } = pageContext;
 
   return (
-    <PageContent width="95%" maxWidth="1000px">
-      <Title>{title}</Title>
+    <PageContent width="95%" maxWidth="1000px" title={title}>
```
Comment (mbwatson) [inline src/templates/latest-updates/index.jsx:21]: same as above

File: src/templates/events/event-template.jsx [hunk 6]
```diff
@@ -141,8 +141,7 @@ const SingleEventTemplate = ({ data, pageContext, children }) => {
   const past = dateString > eventDate
 
   return (
-    <PageContent width="95%" maxWidth="1000px">
-      <Title>{title}</Title>
+    <PageContent width="95%" maxWidth="1000px" title={title}>
```
Comment (mbwatson) [inline src/templates/events/event-template.jsx:144]: can omit `width="95%"`

File: src/templates/events/past-events-template.jsx (hunk 1 above)
Comment (suejinkim20) [reply src/templates/events/past-events-template.jsx:18]: ah yes, thank you!

**Review by mbwatson — APPROVED** [review]
:+1:

**Pull request merged**